src/generator/sfdx-generator.ts

```diff
@@ -194,7 +197,13 @@ export class Generator {
   }
 
   private addTemplateHelper(element: Object): any {
-    return _.extend({}, element, this.templateHelpers);
+    return _.extend({ cliVersion: this.cliVersion }, element, this.templateHelpers);
+  }
+
+  private async initializeCliVersion() {
+    const unformattedCliVersion = await this.runCommand("--version");
+    // Format by removing carriage returns
+    this.cliVersion = unformattedCliVersion.replace(/\n/g, "");
```

Comment:
If I recall, `sfdx --version` is shooting a string like `sfdx-cli/6.37.0-2d5d7e1522 (windows-x64) node-v8.9.4` is that too much ?

Reply:
Yes, the version will output a similar string, but I prefer letting the users parse this the way they want in their templates.